tests/apis/platformvm/api.test.ts

```diff
@@ -2422,8 +2428,8 @@ describe("PlatformVMAPI", (): void => {
 
       expect(intotal.toString(10)).toBe("4000000000")
       expect(outtotal.toString(10)).toBe("1000000000")
-      expect(staketotal.toString(10)).toBe("3000000000")
-      expect(totaltotal.toString(10)).toBe("4000000000")
+      //expect(staketotal.toString(10)).toBe("3000000000")
```

Comment:
Same with this comment

Reply:
Should remain to replace them with LockedIn / LockedOut logic as soon they are in.